### ISSUE TITLE
A lot of log files might be pulled on errors

### ISSUE THREAD

**Issue by Fobhep** [confluentinc/cp-ansible#688]
First up: The new "get log back to ansible controller" feature is really awesome - thanks a lot :)

However one might end up with quite a bunch of logs on the controller.
Do you think a possible feature would be to record the timestamp of the deployment start and pull only logs that are younger than that?
Maybe that could be called `slim_logging` :)

Also eg I am not sure if I really need all types of kafka logs (requests, authorizer etc) for debugging a failed deployment

**Comment by domenicbove** [comment]
Yes this is a great thought, I recently was debugging something w a user and we had to remove some kafka log files because he could not email the entire tar. At least we tar it haha. I think the GC logs aren't helpful/ waste space.

Do you have the full contents of one of these tars that you can paste? Lets try and pin point the "non-useful" files!

**Comment by Fobhep** [comment]
sure - I will get back at you asap with that

**Comment by utkarsh5474** [comment]
Hey @Fobhep 
We would love to get such a list if you get time.
